tests/unit/components/Results/RecordStatus.spec.js

```diff
@@ -0,0 +1,35 @@
+import {shallowMount} from "@vue/test-utils";
+import Vuetify from "vuetify"
+import RecordStatus from "@/components/Results/RecordStatus.vue"
+import light from '@/plugins/theme'
+
+
+let vuetify = new Vuetify({
+  theme: {
+    themes: {light}
+  }
+});
+
+describe("RecordStatus.vue", function () {
+  let wrapper;
+
+  wrapper = shallowMount(RecordStatus, {
+    vuetify,
+    propsData: {record: {status: 'ready', type: 'collection'}}
+  });
+
+  it("can be instantiated", () => {
+    expect(wrapper.name()).toMatch("RecordStatus");
```

Comment:
Please use ```wrapper.vm.$options.name``` instead of ```wrapper.name()``` as later one is deprecated and it is giving console error while running test